assets/app/views/newfromtemplate.html

```diff
@@ -1,37 +1,40 @@
-<div class="container">
-  <div class="row">
-    <div class="col-md-12">
-      <div ng-controller="NewFromTemplateController" class="create-from-template">
-        <div class="row">
-          <div class="col-md-2 template-name">
-            <span class="fa fa-cubes hidden-xs hidden-sm"></span>
-          </div>
-          <div class="col-md-9">
-            <osc-image-summary resource="template"></osc-image-summary>
-            <div ng-if="templateImages.length" class="images gutter-bottom">
-              <h2>Images</h2>
-              <ul class="list-unstyled" ng-repeat="image in templateImages">
-                <li>
-                  <i class="fa fa-cube"></i>
-                  <span class="name">{{ image.name }}</span>
-                </li>
-              </ul>
-            </div>
-            <labels labels="template.labels"></labels>
-            <template-options></template-options>
-            <div class="project gutter-bottom">
-            Items will be created in the <b>{{ projectDisplayName() }}</b> project.
+<div ng-controller="ProjectController">
```

Comment:
`ProjectController` is needed for `<return-to-overview project="project">`. Unfortunately it adds a level of indentation, creating a large diff.